examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,65 @@
+"""
+=========================
+Zoomed-in cutout of a Map
+=========================
+
+This example demonstrates how to showcase a particular region of a full disk
+image within the same figure.
+
+This is a useful for figures that want to provide an overview of your subject.
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.patches import ConnectionPatch
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# We will start off by defining a region of interest that we want to focus on.
+# In this case, we want to focus on the center of the Sun.
+length = 200 * u.arcsec
+x0 = -50 * u.arcsec
+y0 = -30 * u.arcsec
```

Comment:
```suggestion
length = 400 * u.arcsec
x0 = -300 * u.arcsec
y0 = -100 * u.arcsec
```

Currently, the cutout is not centered on any particular region of interest. This (approximately) centers the FOV on the active region